docs/en/contributing/working.md

```diff
@@ -61,11 +29,10 @@ The above outlines the format for the **final** commit, but contributors have ma
 Typical interim commits may look like the following:
 
 ```terminal
-5ba3db6 Update toggle markup for accessibility
-84564a0 Add MSFT styles
+5ba3db6 update toggle markup for accessibility
+84564a0 add border styles
 e142fd1 Rebase with master
-887815f Remove toggle underline on hover
-ac8326d Fix ts-lint errors
+887815f remove toggle underline on hover
 ```
 
 As a core philosophy, we prefer [rebasing over merging](https://www.atlassian.com/git/tutorials/merging-vs-rebasing).
```

Comment:
This seems soft, I prefer the language we used on the CONTRIBUTING page.

"Before submitting a pull request, [rebase](https://www.atlassian.com/git/tutorials/merging-vs-rebasing) your branch from master. Do not use ``git merge`` or the *merge* button provided by Github."